webnode/package.json

```diff
@@ -100,8 +100,7 @@
     "yarn-or-npm": "^2.0.4"
   },
   "scripts": {
-    "start": "cross-env NODE_PATH=src node scripts/start.js",
-    "start-webpack": "NODE_ENV=development webpack-dev-server --config='config/webpack.config.dev.js'",
+    "start": "NODE_ENV=development webpack-dev-server --config='config/webpack.config.dev.js'",
```

Comment:
Did you happen to try this while leaving cross-env in?  The reason cross-env was added iirc is because it allowed both windows-using devs and apple-using devs to runt he script.  If it won't work if you leave 'cross-env' in, then nevermind, but if it will still work it would be better to leave it.

Reply:
can you try to run it on your machine? I dont think we are using anything that cant be used on both.

Reply:
I get an error when I remove cross-env locally.  Granted, that's with the version that's currently in master.  Does leaving cross-env in break anything for you?